Training/python/2018v1/Training_v0p1.py

```diff
@@ -253,7 +254,7 @@ def run_training(train_suffix, model_name, model, data_loader, is_profile):
 
 
 config   = os.path.abspath( "../../configs/training_v1.yaml")
-scaling  = os.path.abspath("../../configs/scaling_params_v1.json")
+scaling  = os.path.abspath("/nfs/dust/cms/user/mykytaua/dataDeepTau/DeepTauTraining/ShuffleMergeSpectral_TrainingScaling/ShuffleMergeSpectral_trainingSamples-2_files_0_50.json")
```

Comment:
if scaling is updated, add it to the repository, don't refer to the files on nfs.